anyway/infographics_utils.py

```diff
@@ -733,11 +708,40 @@ def create_infographics_data(news_flash_id, number_of_years_ago):
     )
     output["widgets"].append(accident_count_by_driver_type.serialize())
 
-    return Response(json.dumps(output, default=str), mimetype="application/json")
+    return json.dumps(output, default=str)
+
+
+get_infographics_data_executor = ThreadPoolExecutor(max_workers=1)
+
+
+def get_infographics_data(news_flash_id, years_ago):
+    try:
+        future = get_infographics_data_executor.submit(
+            infographics_data_cache_updater.get_infographics_data_from_cache,
+            news_flash_id,
+            years_ago,
+        )
+        res = future.result(timeout=3)
```

Comment:
Why timeout is needed? Why did you pick 3?

Reply:
The goal is to avoid hanging of the application in case the transaction hangs.
I chose 3 as in most of the cases the query from the cache will return sooner. We can shorten it.

Reply:
OK, let's leave it like this for now